dev-libs/libgrapheme/patches/libgrapheme-2.0.0.patchset

```diff
@@ -0,0 +1,43 @@
+From 1343367e052b83990f55243bec5947c796aff94b Mon Sep 17 00:00:00 2001
+From: Al Hoang <3811822-hoanga@users.noreply.gitlab.com>
+Date: Sat, 8 Oct 2022 11:03:46 +0000
+Subject: unset ldconfig
+
+
+diff --git a/config.mk b/config.mk
+index f32cd2c..6593fc7 100644
+--- a/config.mk
++++ b/config.mk
+@@ -30,5 +30,5 @@ CC       = cc
+ BUILD_CC = $(CC)
+ AR       = ar
+ RANLIB   = ranlib
+-LDCONFIG = ldconfig # unset to not call ldconfig(1) after install/uninstall
++LDCONFIG = 
+ SH       = sh
+-- 
+2.37.3
+
+
+From b9b0fbe7ef6308a528e70fe4cea049fa8e6cb720 Mon Sep 17 00:00:00 2001
+From: Al Hoang <3811822-hoanga@users.noreply.gitlab.com>
+Date: Sat, 8 Oct 2022 16:03:52 +0000
+Subject: update PREFIX
```

Comment:
Why update PREFIX? You can use that as you do with the others like `PREFIX=$prefix`

Reply:
Checked, second part of the patch isn't needed

Reply:
removed patch for PREFIX